tests/Unit/Parallel/CMakeLists.txt

```diff
@@ -110,6 +110,37 @@ function(add_algorithm_test_with_input_file TEST_NAME INPUT_FILE_DIR)
       ENVIRONMENT "ASAN_OPTIONS=detect_leaks=0")
 endfunction()
 
+# Note that we have a 'balance' here that happens 2.5e4 of times per second just
```

Comment:
It depends on the run, there's a lot of noise in how many balances occur -- if you balance much less than this, I found that I would not consistently get the 5 balances I demand in the test. On average, this gives several dozen balances on my machine, it's just the tail of the distribution can get low sometimes. Too much more often balancing than this and the test takes super long because it's balancing period is about as long as the time it takes to balance, so it doesn't get much time to take computational action :P.
I'll add a bit more explanation to the cmake comment to that effect.